src/components/GlobeModiteList/index.tsx

```diff
@@ -8,16 +8,18 @@ import { useData } from '../../service/Data'
 import { ContextArray, defaultViewport, useMap } from '../../service/Map'
 import { Dimensions, useWindowDimensions } from '../../service/WindowDimensions'
 import { DataState } from '../../types/service/Data'
+import useModites from '../../hook/useModites.tsx'
 import s from './styles.module.scss'
 
 const GlobeModiteList = () => {
   const [state]: [DataState] = useData()
   const [viewport, setViewport]: ContextArray = useMap()
   const dimensions: Dimensions = useWindowDimensions()
+  const sortedRecords = useModites([...state.modites])
```

Comment:
what is the benefit of recreating the array reference on every render? I noticed that it's being recreated again inside the hook, too.

Reply:
The records array passed to useModites() shouldn't be modified or sorted.  The caller or someone up the food chain might care about the contents of the list.

The records passed to useModites() may not be the same array each time.  If we implement pull to refresh, for example.

Reply:
I also want to point out that the unmodified list (before sorting) is by last name ascending.  This makes our job a LOT easier because figuring out if a last name is Grisigono or "de Grisisono" or "el Grisigono" or "Grisigono Jr" or "Grisigono IV", and so on.